index.html

```diff
@@ -1345,43 +1345,14 @@ <h2 property="schema:name">Storage</h2>
             <div datatype="rdf:HTML" property="schema:description">
               <p>
                 The <code>pim:storage</code> predicate is used to indicate where
-                a WebID owner stores their data. An app wanting to access the
-                WebID owner's Pod should find its location using the
-                <code>pim:storage</code> predicate. 
+                a WebID owner stores their data. When an application needs to access the
+                WebID owner's storage, it can find its location using the
+                <code>pim:storage</code> predicate.
               </p>
               <p>
                 A Solid Profile SHOULD contain at least one triple with the
                 WebID as subject, <code>pim:storage</code> as predicate, and the
-                root container of the storage as the object. For example:
-              </p>
-              <pre><code>&lt;?WebID&gt; &lt;http://www.w3.org/ns/pim/space#storage&gt; &lt;?StorageSpace&gt;.</code></pre>
-              <p>
-                If no storage is found within the Solid Profile, an app MAY
-                determine the location of a WebID in the context of a WebID
-                Profile Document hosted on a Solid server as per Solid Protocol
-                0.9, but this method is not guaranteed to be usable by the app
-                in cases where the WebID Profile Document is hosted elsewhere.
-              </p>
-              <p>
-                If no storage space is found through either the profile triples
-                or finding the closest storage, an app MAY prompt the user to
-                find a location to access data.
-              </p>
-              <p>
-                A Pod Management App MAY offer to create a triple by prompting
-                the user for the Pod location and desired discoverability of the
-                Pod. For example, Jose might want their Pod only to store
-                private data and not even let anyone discover that they own the
-                Pod. In that case the app should write the
-                <code>pim:storage</code> triple in the
-                <code>pim:preferencesFile</code> rather than in the WebID
-                Profile Document or a publicly available extended profile
-                document.
-              </p>
-              <p>
-                Applications SHOULD NOT depend on guessing a storage location
-                based on the WebID URI because the WebID Profile Document may or
-                may not be on the same server as the Pod.
+                root container of the storage as the object, but MAY contain more than one.
```

Comment:
```suggestion
                storage as the object.
```

The storage URI is the root container of a storage space, so perhaps not necessary to indicate this here.

Saying "at least one triple" already indicates that there could be more than one so "MAY contain more than one" is not necessary.